shared/templates/grub2_bootloader_argument/ansible.template

```diff
@@ -30,7 +30,8 @@
 {{% else %}}
 
 - name: get current kernel parameters
-  command: '/usr/bin/grub2-editenv - list'
+  ansible.builtin.shell:
+    cmd: '/usr/bin/grub2-editenv - list | echo "kernelopts="'
```

Comment:
But this will always print just `kernelopts=` and nothing else, won't it?

Reply:
Thanks for bringing this up @jan-cerny :bow: 

Definitely very stupid mistake, thinking grep , writing echo :). Changed it in ea14ebd